flex-checks/src/main/java/org/sonar/flex/checks/LineLengthCheck.java

```diff
@@ -19,32 +19,29 @@
  */
 package org.sonar.flex.checks;
 
-import com.google.common.io.Files;
 import com.sonar.sslr.api.AstNode;
+import com.sonar.sslr.api.AstNodeType;
+import java.nio.charset.Charset;
+import java.text.MessageFormat;
+import java.util.Collections;
+import java.util.List;
+import javax.annotation.Nullable;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.sonar.check.Priority;
 import org.sonar.check.Rule;
 import org.sonar.check.RuleProperty;
-import org.sonar.flex.api.CharsetAwareVisitor;
+import org.sonar.flex.FlexCheck;
 import org.sonar.flex.checks.utils.Tags;
 import org.sonar.squidbridge.annotations.SqaleConstantRemediation;
-import org.sonar.squidbridge.checks.SquidCheck;
-import org.sonar.sslr.parser.LexerlessGrammar;
-
-import javax.annotation.Nullable;
-import java.io.IOException;
-import java.nio.charset.Charset;
-import java.util.Collections;
-import java.util.List;
 
 @Rule(
   key = "LineLength",
   name = "Lines should not be too long",
   priority = Priority.MINOR,
   tags = Tags.CONVENTION)
 @SqaleConstantRemediation("1min")
-public class LineLengthCheck extends SquidCheck<LexerlessGrammar> implements CharsetAwareVisitor {
+public class LineLengthCheck extends FlexCheck {
 
   private static final int DEFAULT_MAXIMUM_LINE_LENHGTH = 80;
   private static final Logger LOG = LoggerFactory.getLogger(LineLengthCheck.class);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "LOG" private field. ([squid:S1068](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1068))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.flex%3Aflex&branch=SONARFLEX-105&issues=AWGPc9z6Ol9lQhPgXcMv&open=AWGPc9z6Ol9lQhPgXcMv)

[ISSUE_KEY]: AWGPc9z6Ol9lQhPgXcMv